recipes/seqlogo/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "seqlogo" %}
+{% set version = "0.1.12" %}
+
+package:
+  name: "{{ name }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b172a4b52ea406d1b3225e3e6a580e69b435f2e69a9e1e94c68cf982e948ccb7
+
+build:
+  skip: True # [py27]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - numpy
+    - pandas
+    - pytest
+    - python
+    - weblogo
+    - ghostscript
+
+test:
+  imports:
+    - seqlogo
+
+about:
+  home: https://github.com/betteridiot/seqlogo
+  license: BSD
+  license_family: BSD
+  license_file: LICENSE
```

Comment:
This is the path to the licence file. As you have this in your github repo you don't need to put it into this PR. Please remove the file from this PR

Reply:
Thank you for taking care of this